requirements.txt

```diff
@@ -3,7 +3,7 @@
 # This file records the packages and requirements needed in order for
 # pygeoprocessing to work as expected.
 
-GDAL>=2.2.0,<3.0
+GDAL==3.0.4
```

Comment:
I'm sorry I didn't notice this before, we've had problems in the past when we've pegged the version of packages to an *exact* version. Could you change this to `GDAL>=3.0`? Unless there's something in a particular version that you don't want before 3.0.4? Otherwise we'll have a pain if we have a docker image with 3.0.2 that'll need a whole rebuild etc.etc..

Reply:
Hey @richpsharp , There were some bug fixes since GDAL 3.0.2 that have effected the precision of transforms, therefore I was using 3.0.4 because that is what I created my tests in, and at least one of them would otherwise fail in 3.0.2 by a decimal. Let me know if you'd like to talk about this more.
I made the change to `GDAL>=3.0.4` in [83bd15](https://github.com/natcap/pygeoprocessing/pull/31/commits/83bd1546903dfef0eb234fdcb740cb551261e82d)

Reply:
Hi @dcdenu4, unless you think it needs to be 3.0.4 could you change it to `GDAL>=3.0.2`? (I've got a docker image that's using 3.0.2 that'd be a pain to rebuild, but I could if it was necessary).